lib/src/qr_code.dart

```diff
@@ -78,6 +79,12 @@ class QrCode {
     _dataCache = null;
   }
 
+  void addByteData(ByteData data) {
```

Comment:
Would you create a private helper that can be called by both `addData` and `addByteData`?